daprdocs/content/en/operations/monitoring/newrelic.md

```diff
@@ -0,0 +1,347 @@
+
+Automated correlation is part of the visualization capabilities of New Relic.
+
+### Pod-level details
+
+![New Relic K8s Pod Level Details](/images/nr-k8s-pod-level-details.png)
+
+### Logs in Context
+
+![New Relic K8s Logs In Context](/images/nr-k8s-logs-in-context.png)
+
+### Distributed Tracing
+
+![New Relic Distributed Tracing 1](/images/nr-distributed-tracing-1.png)
+
+![New Relic Distributed Tracing 2](/images/nr-distributed-tracing-2.png)
+
+## New Relic APM curated experience
+
+### Front-end application with New Relic language agent
+
+![New Relic APM Frontend](/images/nr-apm-frontend.png)
+
+### .Net Core Subtractor with OpenTelemetry Exporter
+
+![New Relic APM Subtractor 1](/images/nr-apm-subtractor-1.png)
+
+![New Relic APM Subtractor 2](/images/nr-apm-subtractor-2.png)
+
+## New Relic Dashboards
+
+### Kubernetes Overview
+
+![New Relic Dashboard Kubernetes Overview](/images/nr-dashboard-k8s-overview.png)
+
+### Dapr System Services
+
+![New Relic Dashboard Dapr System Services](/images/nr-dashboard-dapr-system-services.png)
+
+### Dapr Metrics
+
+![New Relic Dashboard Dapr Metrics 1](/images/nr-dashboard-dapr-metrics-1.png)
+
+![New Relic Dashboard Dapr Metrics 2](/images/nr-dashboard-dapr-metrics-2.png)
+
+![New Relic Dashboard Dapr Metrics 3](/images/nr-dashboard-dapr-metrics-3.png)
+
+![New Relic Dashboard Dapr Metrics 4](/images/nr-dashboard-dapr-metrics-4.png)
+
+![New Relic Dashboard Dapr Metrics 5](/images/nr-dashboard-dapr-metrics-5.png)
+
+![New Relic Dashboard Dapr Metrics 6](/images/nr-dashboard-dapr-metrics-6.png)
+
+![New Relic Dashboard Dapr Metrics 7](/images/nr-dashboard-dapr-metrics-7.png)
+
+## New Relic Grafana integration
+
+New Relic teamed up with [Grafana Labs](https://grafana.com/) so you can use our [Telemetry Data Platform](https://newrelic.com/platform/telemetry-data-platform) as a data source for Prometheus metrics and see them in your existing dashboards, seamlessly tapping into the reliability, scale, and security provided by New Relic.
+
+[Grafana dashboard templates](https://github.com/dapr/dapr/blob/227028e7b76b7256618cd3236d70c1d4a4392c9a/grafana/README.md) to monitor Dapr system services and sidecars can easily be used without any changes. New Relic provides a [native endpoint for Prometheus metrics](https://docs.newrelic.com/docs/integrations/grafana-integrations/set-configure/configure-new-relic-prometheus-data-source-grafana) into Grafana. A datasource can easily be set-up:
+
+![New Relic Grafana Data Source](/images/nr-grafana-datasource.png)
+
+And the exact same dashboard templates from Dapr can be imported to visualize Dapr system services and sidecars.
+
+![New Relic Grafana Dashboard](/images/nr-grafana-dashboard.png)
+
+## New Relic Alerts
+
+All the data that is collected from Kubernetes, Dapr or any services that run on top of Kubernetes can be used to set-up alerts and notifications into the preferred channel of your choice.
```

Comment:
Please add a "Related links" or "References"  section in the end with links for relevant New Relic docs